clang/lib/CIR/CodeGen/CIRGenCXX.cpp

```diff
@@ -154,7 +155,8 @@ bool CIRGenModule::tryEmitBaseDestructorAsAlias(const CXXDestructorDecl *D) {
   // symbol reference from another TU. The other TU must also mark the
   // referenced symbol as weak, which we cannot rely on.
   if (cir::isWeakForLinker(Linkage) && getTriple().isOSBinFormatCOFF()) {
-    llvm_unreachable("NYI");
+    assert(false && "please sent a PR with a test and remove this.\n");
```

Comment:
Not sure I understand the point here - if the assert is not triggered, it means this codepath is executed, which means the current testcase is enough to prove the point? Just returning true here is all you need. I just noticed I merged your previous PR with the same logic and I missed this, please remove the assert from that one (fine if you include it as part of this PR as well).

Reply:
The assertion should always be triggered. My intention is, when it gets triggered, the reproducer can provide a test case only and remove the assertion. So that it is easier than implementing it slighlty.

Reply:
Please change it to `llvm_unreachable("please sent a PR with a test and remove this");`, let's be consistent with the rest of CIR (as mentioned before)